galata/test/jupyterlab/sidebars.test.ts

```diff
@@ -194,3 +201,157 @@ test.describe('Sidebars', () => {
+
+      await page.keyboard.press('Enter');
+
+      expect(await page.sidebar.isTabOpen(leftSidebarId)).toEqual(true);
+    });
+  });
+
+  rightSidebarIds.forEach(rightSidebarId => {
+    test(`Open Sidebar tab ${rightSidebarId} via keyboard navigation`, async ({
+      page
+    }) => {
+      await page.goto();
+
+      await page.sidebar.close('right');
+
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
+        await page.keyboard.press('Tab');
+        let propertyInspectorIsFocused = await page.evaluate(
+          () => document.activeElement?.getAttribute('data-id')
+        );
+        if (propertyInspectorIsFocused === rightSidebarIds[0]) {
+          break;
+        }
+      }
+
+      // eslint-disable-next-line no-constant-condition
+      while (rightSidebarId !== rightSidebarIds[0]) {
+        await page.keyboard.press('ArrowDown');
+        let IsFocused = await page.evaluate(
+          () => document.activeElement?.getAttribute('data-id')
+        );
+        if (IsFocused === rightSidebarId) {
+          break;
+        }
+      }
+      await page.keyboard.press('Enter');
+
+      expect(await page.sidebar.isTabOpen(rightSidebarId)).toEqual(true);
+    });
+  });
+
+  sessionsAndDebuggerAriaLabels.forEach(sessionsAndDebuggerAriaLabel => {
+    test(`Open accordion panels ${sessionsAndDebuggerAriaLabel} via keyboard navigation`, async ({
+      page
+    }) => {
+      await page.goto();
+
+      await page.sidebar.openTab('jp-running-sessions');
+      await page.sidebar.openTab('jp-debugger-sidebar');
+
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
+        await page.keyboard.press('Tab');
+        let IsFocused = await page.evaluate(
+          () => document.activeElement?.getAttribute('aria-label')
+        );
+        if (IsFocused === sessionsAndDebuggerAriaLabel) {
+          break;
+        }
+      }
+
+      await page.keyboard.press('Enter');
+
+      const isExpanded = await page.evaluate(
+        () => document.activeElement?.getAttribute('aria-expanded')
+      );
+
+      expect(isExpanded).toBeTruthy();
+    });
+  });
+
+  propertyElementAriaLabels.forEach(propertyElementAriaLabel => {
+    test(`Open accordion panels ${propertyElementAriaLabel} via keyboard navigation`, async ({
+      page
+    }) => {
+      await page.goto();
+
+      await page.sidebar.openTab('extensionmanager.main-view');
+
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
+        await page.keyboard.press('Tab');
+        let IsFocused = await page.evaluate(
+          () => document.activeElement?.getAttribute('aria-label')
+        );
+        if (IsFocused === propertyElementAriaLabel) {
+          break;
+        }
+      }
+
+      await page.keyboard.press('Enter');
+
+      const isExpanded = await page.evaluate(
+        () => document.activeElement?.getAttribute('aria-expanded')
+      );
+
+      expect(isExpanded).toBeTruthy();
```

Comment:
We should expect `isExpanded` to be false, the accordion is opened by default, and close when pressing `entrer` the first time.
`isExpanded` here is a string containing 'false', the test pass because it is not a empty string.

For more robustness we could check the initial state, press enter, check that the state changed, and do it again (to ensure opening and closing works well on enter).